tests/zenko_tests/node_tests/backbeat/ReplicationUtility.js

```diff
@@ -448,52 +497,66 @@ class ReplicationUtility {
     }
 
     createBucket(bucketName, cb) {
-        this.s3.createBucket({ Bucket: bucketName }, cb);
+        this.s3.send(new CreateBucketCommand({ Bucket: bucketName }))
+            .then(data => cb(null, data))
+            .catch(cb);
     }
 
     createVersionedBucket(bucketName, cb) {
         return async.series([
-            next => this.s3.createBucket({ Bucket: bucketName }, next),
-            next => this.s3.putBucketVersioning({
+            next => this.s3.send(new CreateBucketCommand({ Bucket: bucketName }))
+                .then(() => next())
```

Comment:
```suggestion
                .then(next)
```

But I guess it's because you don't want to send the result of the create bucket. To make it explicit, you can write `.then((_data) => next())`. It's nit, not sure what the best here